src/applicator.ts

```diff
@@ -138,28 +137,28 @@ function applyMiddlewareToType<TSource, TContext, TArgs>(
 
   if (isMiddlewareFunction(middleware)) {
     const resolvers = Object.keys(fieldMap).reduce(
-      (resolvers, fieldName) => ({
-        ...resolvers,
-        [fieldName]: applyMiddlewareToField(
+      (resolvers, fieldName) => {
+        resolvers[fieldName] = applyMiddlewareToField(
           fieldMap[fieldName],
           options,
           middleware as IMiddlewareFunction<TSource, TContext, TArgs>,
-        ),
-      }),
+        );
+        return resolvers;
+      },
       {},
     )
 
     return resolvers
   } else {
     const resolvers = Object.keys(middleware).reduce(
-      (resolvers, field) => ({
-        ...resolvers,
-        [field]: applyMiddlewareToField(
+      (resolvers, fieldName) => {
+        resolvers[fieldName] = applyMiddlewareToField(
           fieldMap[field],
           options,
           middleware[field],
-        ),
-      }),
+        );
```

Comment:
`Cannot find name 'field'`. Could you please rename `fieldName` back to `field`, or use `fieldName` inside the function?